Tella/Scenes/Uwazi/Views/Entity/Widgets/UwaziRelationshipWidget.swift

```diff
@@ -0,0 +1,86 @@
+//
+//  UwaziRelationshipWidget.swift
+//  Tella
+//
+//  Created by gus valbuena on 4/10/24.
+//  Copyright © 2024 HORIZONTAL. All rights reserved.
+//
+
+import SwiftUI
+
+struct UwaziRelationshipWidget: View {
+    @ObservedObject var prompt: UwaziRelationshipEntryPrompt
+    @EnvironmentObject var entityViewModel: UwaziEntityViewModel
+    var body: some View {
+        VStack {
+            Text(LocalizableUwazi.uwaziEntityRelationshipExpl.localized)
+                .font(.custom(Styles.Fonts.regularFontName, size: 12))
+                .foregroundColor(Color.white.opacity(0.87))
+                .frame(maxWidth: .infinity, alignment: .leading)
+            Button {
```

Comment:
What do you think if you put 

```
            Button {
                navigateTo(destination: EntitySelectorView(selectedValues: $prompt.value)
                    .environmentObject(entityViewModel)
                    .environmentObject(prompt)
                )
            } label: {
                entitiesSelect
            }
            .background(Color.white.opacity(0.08))
            .cornerRadius(15)

```
in a variable `selectEntitiesButton`

    var selectEntitiesButton: some View {
        Button {
            navigateTo(destination: EntitySelectorView(selectedValues: $prompt.value)
                .environmentObject(entityViewModel)
                .environmentObject(prompt)
            )
        } label: {
            selectEntitiesView
        }
        .background(Color.white.opacity(0.08))
        .cornerRadius(15)

    }